digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/controller/UtilityApiDelegateImpl.java

```diff
@@ -8,34 +9,55 @@
 import org.springframework.stereotype.Service;
 
 import ca.bc.gov.open.digitalformsapi.viirp.api.UtilityApiDelegate;
-import ca.bc.gov.open.digitalformsapi.viirp.model.VipsPingServiceResponse;
+import ca.bc.gov.open.digitalformsapi.viirp.exception.DigitalFormsException;
+import ca.bc.gov.open.digitalformsapi.viirp.model.PingResponse;
 import ca.bc.gov.open.jag.ordsvipsclient.api.HealthApi;
 import ca.bc.gov.open.jag.ordsvipsclient.api.handler.ApiException;
 
+/**
+ * 
+ * Utility Delegate. ORDS Ping service, etc.  
+ * 
+ * @author 176899
+ *
+ */
 @Service
 public class UtilityApiDelegateImpl implements UtilityApiDelegate{
 	
 	private final Logger logger = LoggerFactory.getLogger(UtilityApiDelegateImpl.class);
 	
 	@Autowired
 	private HealthApi vipsHealthApi;
+	
+	@Autowired
+	private ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.api.HealthApi digitalFormsHealthApi;
 		
+	@SuppressWarnings("unchecked")
 	@Override
-	public ResponseEntity<VipsPingServiceResponse> utilityVipsordsPingCorrelationIdGet(String correlationId) {
+	public ResponseEntity<PingResponse> utilityOrdsPingCorrelationIdGet(String correlationId) {
 		
-		logger.info("Heard a call to the endpoint 'utilityVipsordsPingCorrelationIdGet'");
+		logger.info("Heard a call to 'utilityOrdsPingCorrelationIdGet'");
 		
-		VipsPingServiceResponse resp = new VipsPingServiceResponse();
+		PingResponse resp = new PingResponse();
 		try {
-			resp.setStatus(vipsHealthApi.health().getStatus()); 
-			resp.setHost(vipsHealthApi.health().getHost());
-			resp.setInstance(vipsHealthApi.health().getInstance()); 
+			
+			JSONObject jsonResponse = new JSONObject();
+			jsonResponse.put("VIPS ORDS Health Status", vipsHealthApi.health().getStatus());
+			jsonResponse.put("DIGITAL FORMS ORDS Health Status", digitalFormsHealthApi.health().getStatus());
+			resp.setResponseMessage(jsonResponse);
+			
+			return new ResponseEntity<>(resp, HttpStatus.OK); 
 						
 		} catch (ApiException e) {
-			System.out.println("Error: " + e.getMessage());
+			String msg = "Error encountered while pinging VIPS ORDS Health endpoint: " + e.getMessage();
 			e.printStackTrace();
```

Comment:
I think we should use the logging framework instead of the printStackTrace() for better logging.
```suggestion
			logger.error(msg, e);
```

Reply:
Yes, you're absolutely correct. An oversight on my part for leaving out the logger.error. But, I like to leave in the e.printStackTrace because when things go wrong it often provides a much more in-depth explanation of why the error occurred.